documentation/typespec-end-to-end-scenarios.md

```diff
@@ -68,57 +68,65 @@ classDef grey fill:#CCCCCC,color:#555555;
 User((::))-->A
 A["clone Rest-API and SDK repo locally"] --> B
 B["2.1 pre-requsite dependencies installation
-(each language would have a installation script)"] --> C
-C["2.2 TypeSpec-Project-Sync.ps1
-(a.create tsp-location.yaml and drop to temp location
-b.update tsp-location.yaml if existed
-c.fetch specs from remote spec repo or use local spec repo)"] --> D
- D["2.3 TypeSpec-Project-Generate.ps1
-(a.create scaffolding for new project
-b.update tsp-location.yaml
-c.copy typespec specs to temp folder
-d.generate sdk code)"]
+(each language would have a installation script)"] --> F
+F["2.2 TypeSpec-Project-Prcoess.ps1"] --> C
```

Comment:
Is there a way to put the sync and generate as sub-boxes inside of the process box so it is clear they run as part of the script instead of needing to be ran independently.

Reply:
Updated.